### PR TITLE
Portfolio: desktop version

### PR DESCRIPTION
**Pull request by Ibtisam34**
For the fourth project in my  portfolio website, I have:

- [ ] Use media queries to make sure that my website displays correctly on desktop screens.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,313 +1,351 @@
...
+            name=""
+            id=""
+            cols="30"
+            rows="10"
+            type="text"
+            placeholder="Enter text here"
+            maxlength="500"
+            required
+          ></textarea>
+          <button class="button" type="submit">Get in touch</button>
```
Comment (ichala) [inline index.html]: - [ ] to match Figma design this button should be aligned to the left , kindly fix that

image

File: index.html [hunk 2]
```diff
@@ -1,313 +1,351 @@
...
+            <h1 class="myskillHeading">Skills</h1>
+          </div>
+          <ul class="skill-List">
+            <li>Codekit</li>
+            <li>Github</li>
+            <li>Gitlab</li>
+            <li>Terminal</li>
+            <li>Codepen</li>
+          </ul>
+        </div>
```
Comment (ichala) [inline index.html:261]: - [ ] To match Figma design kindly fix these issues : 

1. Kindly remove the dots before each technologies item 
2. kindly make sure that the technologies lists are aligned to the left 


Figma : 

image

Your Design:

image

File: index.html [hunk 3]
```diff
@@ -1,313 +1,351 @@
...
+                <li>bootstrap</li>
+                <li>Ruby</li>
               </ul>
-              <button type="button">See Project</button>
             </div>
-          
-            <div class="work1" id="w4">
-              <h1>Profesional Art Printing Data</h1>
+            <button class="button" id="w3button">See Project</button>
+          </div>
```
Comment (ichala) [inline index.html]: - [ ] To match Figma design kindly fix these issues in this section : 

1. kindly add the correct margins & padding to the content of the cards shown in the image below 
2. kindly remove the border of the technologies squares/buttons

your design : 
image


Figma Design : 
image

File: index.html [hunk 4]
```diff
@@ -1,313 +1,351 @@
...
+            <div class="work1Discription" id="w3work1Discription">
+              <h2>
+                <span id="w3b">Profesional Art Printing Data</span>
+                <span id="w3d">Website Protfolio </span>
+              </h2>
               <p>
-                A daily selection of privately personalized reads; no accounts or
-                sign-ups required. has been the industry's standard
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required. has been the industry's standard
```
Comment (ichala) [inline index.html:144]: - [ ] there's a small issue with the second card in your `work-cards` section, all the cards are working fine when I hover, but only the second one is not hiding the content when I hover, kindly fix that

image

**Review by ichala — CHANGES_REQUESTED** [review]
Hi @Ibtisam34 , 

Good job so far! 👏👏👏 
There are some issues that you still need to work on to go to the next project but you're almost there!

the-simpsons-homer-simpson

## To Highlight 🎉

- Good Design✔️ 
- No linters Errors ✔️


 ## Required Changes ♻️

- [ ] **_Check The Comments below_**

## Optional suggestions

- N/A


**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Ibtisam34** [comment]
If you don't mind may you assist me the button taking to left its
disturbing me plz

**Comment by Ibtisam34** [comment]
Kindly  ali assist me today is the last day 🙏 plz come and assist me with
this errors

**Comment by ichala** [comment]
@Ibtisam34 tag me please next time so i can get your messages are you available on slack?

**Review by BertrandConxy — APPROVED** [review]
Hi @Ibtisam34, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 💪 
- No linter errors
- project requirements are met
- project works fine
- good PR description
- correct flow is used

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @BertrandConxy in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**